kivy/core/window/_window_sdl2.pyx

```diff
@@ -507,6 +507,16 @@ cdef class _WindowSDL2Storage:
                 windows_info.hdc = wm_info.info.win.hdc
                 return windows_info
 
+    def get_native_handle(self):
+        window_info = self.get_window_info()
+
+        IF USE_WAYLAND:
```

Comment:
We're trying to avoid `IF/ELSE` and `DEF` on new code (and slowly migrating during changes required for `3.0.0`) as is deprecated.

See: http://docs.cython.org/en/latest/src/userguide/migrating_to_cy30.html#deprecation-of-def-if